tests/test_integration.py

```diff
@@ -4,19 +4,19 @@
 from exchange.message import Message
 from exchange.moderators import ContextTruncate
 from exchange.providers import get_provider
-from exchange.providers.ollama import OLLAMA_MODEL
```

Comment:
curious why this is removed? main reason this is here was to test the model (at least ad-hoc) with what we suggest in configuration.

Reply:
Fixed